azurelinuxagent/common/agent_supported_feature.py

```diff
@@ -58,15 +59,35 @@ def __init__(self):
                                                   supported=_MultiConfigFeature.__SUPPORTED)
 
 
+class _ETPFeature(AgentSupportedFeature):
+
+    __NAME = SupportedFeatureNames.ExtensionTelemetryPipeline
+    __VERSION = "1.0"
+    __SUPPORTED = True
+
+    def __init__(self):
+        super(_ETPFeature, self).__init__(name=self.__NAME,
+                                          version=self.__VERSION,
+                                          supported=self.__SUPPORTED)
+
+
 __CRP_SUPPORTED_FEATURES = {
-    CRPSupportedFeatureNames.MultiConfig: _MultiConfigFeature()
+    SupportedFeatureNames.MultiConfig: _MultiConfigFeature()
+}
+
+
+__EXTENSION_SUPPORTED_FEATURES = {
```

Comment:
I would just add a comment in these 2 lists saying the first/second one is the list of features that the agent supports and we advertise to CRP/Extensions.

From the variable names it reads like it is CRP/Extensions who support the features.

Reply:
Done